extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-hive-metastore-service-chart/values.yaml

```diff
@@ -2,8 +2,8 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 app:
-  # Sets labels for app.kubernetes.io/name Default is Chart.name (aissemble-hive-metastore-service-chart)
-  name: ''
+  # Sets labels for app.kubernetes.io/name. Default is Chart.name (aissemble-hive-metastore-service-chart)
+  name: 'hive-metastore-service'
```

Comment:
A: I think the previous behavior makes more sense in the context of a downstream project, where the chart will naturally inherit the `appName` set in the deploy POM.

Reply:
Oh right. Hmmmm. Ok.

Reply:
I do want to raise one interesting thing that this hints at: The hive charts and STS/SHS charts _feel_ different to interact with from a downstream user perspective.  This is fundamentally because they were built by different people at different times.  If we want to do a more wholesale unification of the styles and approaches to each, _we can do that_.  I wouldn't necessarily do it right now, but I think it's one of those oddities where we'll probably come back to it in the future.  Even basic things like how `metastore-config.xml` is built versus how `hive-site.xml` is loaded into STS.  One takes a raw XML string, while the other takes a list of objects that then gets transformed into an XML string.  This naming difference is sort-of one of those oddities-- Though the main motivator for keeping the name _specifically_ as-is was to minimize the breaking changes from previous versions.